dex/src/main/scala/com/wavesplatform/dex/settings/ConfigExtraOps.scala

```diff
@@ -0,0 +1,15 @@
+package com.wavesplatform.dex.settings
+
+import java.util.Properties
+
+import com.typesafe.config.Config
+
+final class ConfigExtraOps(val self: Config) extends AnyVal {
```

Comment:
Please merge this into already existed [ConfigOps](https://github.com/wavesplatform/dex/blob/DEX-355-dex-grpc-separation/dex/src/main/scala/future/com/wavesplatform/settings/utils/ConfigOps.scala)

Reply:
There are several ConfigOps: in NODE project and in DEX.
I can't add this method to the NODE's project. 

In the other side we have a package "future" in DEX that contains ConfigOps. We need to move its contents outside the "future" package, because it was a hack: this package contained a modern functionality from the NODE's project that we couldn't access to (because of a version). So, to use a new functionality we had to just add the "future." prefix into imports. Now:
1. ConfigOps in the NODE doesn't contain all required functionality. It looks like it was removed
2. We will move from ficus to pureconfig to do same things without pain.

So, I created a separate implicit class for general purpose functions.